### ISSUE TITLE
Copy message from other channel

### ISSUE THREAD

**Issue by ashkufaraz** [zyberspace/php-telegram-cli-client#28]
Hi
Sorry,I have a question about your project(telegram).
Can I get the message channels that I'm not their admin?
I want write a script that read message from other chanel and write to
my chanel.
I can do it?
best regards
thanks

**Comment by ingria** [comment]
No, you cannot: this repo doesn't have any channels-related commands.
